package.json

```diff
@@ -28,6 +28,10 @@
     {
       "name": "Stijn de Witt",
       "email": "StijnDeWitt@hotmail.com"
+    },
+    {
```

Comment:
sorry if i am asking too much, can you also add an example page to gh-pages branch once this PR is merged?